src/neo_stocktwits_data_extractor.py

```diff
@@ -139,7 +138,7 @@ def main():
             print('table laid')
 
             if data:
-                if not os.path.isfile('../data/raw/'+optionalArgs.name):
+                if optionalArgs.overwrite or not os.path.isfile('../data/raw/'+optionalArgs.name):
                     df.to_csv('../data/raw/'+optionalArgs.name, index=False, encoding='utf-8')
                 else: # else it exists so append without writing the header
                     df.to_csv('../data/raw/'+optionalArgs.name, mode='a', header=False, index=False)
```

Comment:
Will this work correctly if i run with option -o, but specify a file that doesn't exist yet?

Reply:
Yeah. The logic is "if overwrite is true, or if the file doesn't exist, create the file" (which would overwrite by default) and then "else append to the file"